docker/Jenkinsfile-build-docker

```diff
@@ -1,7 +1,7 @@
 @Library('test-shared-library') _
 import ai.h2o.ci.buildsummary.StagesSummary
 
-DEFAULT_LABEL = 'mr-0xc10'
+DEFAULT_LABEL = 'mr-0x2'
```

Comment:
why? also probably unrelated o this change

Reply:
hmm, will revert, the change was originally made by @michal-raska in original PR, but may have been a temporary change.